Sources/View/Widget/AbstractWidget.swift

```diff
@@ -23,6 +23,10 @@ import UIKit
 class AbstractWidget: UIStackView, UITextFieldDelegate {
     var field: HyperwalletField!
 
+    var isEditable: Bool {
```

Comment:
This should go inside SelectionWidget.

Reply:
it is used in TextWidget as well.

Reply:
TextWidget should directly use isEditable from field. TextWidget has Field. Only for selection widget add that logic inside SelectionWidget

Reply:
`Field` property is in AbstractWidget and all descendant widgets have access to it. Why we should do copy/paste logic?

Reply:
That's the point Field property is in AbstractWidget and all the descendant widgets have access to it so why create another variable for every cell when we can directly check isEditable from field in the descendant widgets?